src/test/ui/moves/issue-46099-move-in-macro.stderr

```diff
@@ -0,0 +1,14 @@
+error[E0382]: use of moved value: `b`
+  --> $DIR/issue-46099-move-in-macro.rs:14:12
+   |
+LL |     let b = Box::new(true);
+   |         - move occurs because `b` has type `std::boxed::Box<bool>`, which does not implement the `Copy` trait
+LL |     test!({b});
+   |            ^
+   |            |
+   |            value moved here
+   |            value used here after move
```

Comment:
Is there a reason this is not formatted as

```suggestion
LL |     test!({b});
   |            ^ value moved here
   |            | value used here after move
```

It's outside the scope of this PR, just wondering.

Reply:
I'm not sure - I think that might just be how the diagnostic framework formats messages.